app/assets/javascripts/training/components/training_module_handler.jsx

```diff
@@ -39,24 +39,12 @@ const TrainingModuleHandler = createReactClass({
       );
     }
     );
-    let moduleSource;
-    if (this.props.training.module.wiki_page) {
-      moduleSource = (
-        <div className="training-module-source">
-          <a href={`https://meta.wikimedia.org/wiki/${this.props.training.module.wiki_page}`} target="_blank">{I18n.t('training.view_module_source')}</a>
-          <br />
-          <a href={`/reload_trainings?module=${this.props.training.module.slug}`}>{I18n.t('training.reload_from_source')}</a>
```

Comment:
I think these messages are still relevant, they just shouldn't be shown until after the data has been loaded. If any empty response has already been received from the server, at that point we should show relevant 'missing' messages.